rmw_cyclonedds_cpp/CMakeLists.txt

```diff
@@ -32,7 +32,12 @@ find_package(ament_cmake_ros REQUIRED)
 find_package(rcutils REQUIRED)
 
 find_package(cyclonedds_cmake_module REQUIRED)
-find_package(CycloneDDS REQUIRED CONFIG)
+find_package(CycloneDDS QUIET CONFIG)
+if (NOT CycloneDDS_FOUND)
+  message(WARNING "Could not find cyclonedds - skipping '${PROJECT_NAME}'")
```

Comment:
Please use the proper case for the name and include the company name: `ADLINK CycloneDDS`.